Tasks/XamarinAndroid/task.json

```diff
@@ -54,35 +53,48 @@
             "name": "outputDir",
             "type": "string",
             "label": "Output Directory",
-            "defaultValue": "$(Agent.BuildDirectory)\\bin\\Release",
+            "defaultValue": "",
```

Comment:
The template uses $(Agent.BuildDirectory)/bin/Release but with the task itself it doesn't make much sense since we don't know in what context the user is adding the task. The xyz directory works in many cases.